marketplace/wpp_templates/models.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+
+from django.db import models
+
+from rest_framework.exceptions import ValidationError
+
+
+class TemplateMessage(models.Model):
+    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
+    name = models.CharField(max_length=60)
+
+    CATEGORY_CHOICES = (
```

Comment:
Constants must be at the top of the class
this works for such lines:
- [32](https://github.com/weni-ai/weni-integrations-engine/pull/166/files#diff-73c4599f012834487ca0855b3d039688b8318535f13ad788afc59f9fab46c419R32)
- [48](https://github.com/weni-ai/weni-integrations-engine/pull/166/files#diff-73c4599f012834487ca0855b3d039688b8318535f13ad788afc59f9fab46c419R48)
- [77](https://github.com/weni-ai/weni-integrations-engine/pull/166/files#diff-73c4599f012834487ca0855b3d039688b8318535f13ad788afc59f9fab46c419R77)
- [103](https://github.com/weni-ai/weni-integrations-engine/pull/166/files#diff-73c4599f012834487ca0855b3d039688b8318535f13ad788afc59f9fab46c419R103)